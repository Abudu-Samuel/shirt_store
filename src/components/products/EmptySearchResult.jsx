import React from "react";

export default ({ query }) => (
  <div className="text-center">
    <h5 className="grey-text mb-4 mt-4">0 products found</h5>
    <h6 className="mb-4 grey-text">
      Unfortunately there was no match found for{" "}
      <span className="field-error">{query}</span>, please try again
    </h6>
    <h6 className="grey-text">
      Sorry, the page you have requested was not found. Try searching for a
      brand or keyword.
    </h6>
  </div>
);
