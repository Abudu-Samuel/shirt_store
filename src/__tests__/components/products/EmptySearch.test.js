import React from "react";
import {
  shallow
} from "enzyme";
import toJSON from "enzyme-to-json";
import EmptySearchResult from "../../../components/products/EmptySearchResult";

describe("<EmptySearchResult />", () => {
  it("renders without crashing", () => {
    const wrapper = shallow( < EmptySearchResult / > );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});