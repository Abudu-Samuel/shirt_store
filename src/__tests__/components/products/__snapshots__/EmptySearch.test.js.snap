// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptySearchResult /> renders without crashing 1`] = `
<div
  className="text-center"
>
  <h5
    className="grey-text mb-4 mt-4"
  >
    0 products found
  </h5>
  <h6
    className="mb-4 grey-text"
  >
    Unfortunately there was no match found for
     
    <span
      className="field-error"
    />
    , please try again
  </h6>
  <h6
    className="grey-text"
  >
    Sorry, the page you have requested was not found. Try searching for a brand or keyword.
  </h6>
</div>
`;
